tools/topology/m4/tdfb.m4

```diff
@@ -57,8 +57,16 @@ define(`W_TDFB',
 `		"'N_TDFB($1)`_data_str"'
 `		"'N_TDFB($1)`_data_str_type"'
 `	]'
-`	bytes ['
+`	mixer ['
```

Comment:
nice!

Reply:
I recall @juimonen suggested to leave the binary control as last.